api/src/main/java/io/strimzi/api/kafka/model/KafkaSpec.java

```diff
@@ -99,6 +102,15 @@ public void setClientsCa(CertificateAuthority clientsCa) {
         this.clientsCa = clientsCa;
     }
 
+    @Description("Configuration maintenance tasks schedule.")
```

Comment:
It's not a schedule. And you should also describe what tasks are considered maintenance tasks.

Reply:
So just mentioning the certificate renewal process?